### ISSUE TITLE
When updating Kendo Theme Bootstrap from 3.2.0 to 3.2.2. Something in dataviz module not happy.

### ISSUE THREAD

**Issue by jcastillo-ait-asia** [telerik/kendo-themes#625]
## I'm submitting a...
* Bug report 

## Current behavior
Module build failed at
```
ERROR in ./src/styles.scss (/Users/johnk/git/uber/uber-spa/node_modules/@angular-devkit/build-angular/src/angular-cli-files/plugins/raw-css-loader.js!/Users/johnk/git/uber/uber-spa/node_modules/postcss-loader/src??embedded!/Users/johnk/git/uber/uber-spa/node_modules/sass-loader/lib/loader.js??ref--14-3!./src/styles.scss)
Module build failed (from /Users/johnk/git/uber/uber-spa/node_modules/sass-loader/lib/loader.js):

            top: calc( #{($padding-y-sm*2) + ($line-height*$font-size)} + 1px);
                                                                             ^
      Incompatible units: 'px' and 'rem'.
      in /Users/johnk/git/uber/uber-spa/node_modules/@progress/kendo-theme-bootstrap/modules/@progress/kendo-theme-default/scss/dataviz/_layout.scss (line 350, column 79)
```
## Environment
styles.scss:

```
@import 'variables';
@import '../../../node_modules/npm-font-open-sans/open-sans.scss';
@import '../../../node_modules/bootstrap/scss/bootstrap.scss';
@import '../../../node_modules/@ngx-share/button/themes/outline/outline-theme';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/calendar';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/datetime';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/listbox';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/dialog';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/window';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/dropdownlist';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/dropdowngrid';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/dropdowntree';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/multiselect';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/switch';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/slider';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/dataviz';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/grid';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/upload';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/autocomplete';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/menu';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/scheduler';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/button';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/popup';
@import '../../../node_modules/@progress/kendo-theme-bootstrap/scss/input';
@import 'mixin';

**Comment by joneff** [comment]
By default, both the font-size and padding are in the same units. Perhaps, you have customized either?

Never the less this is a valid bug report

**Comment by joneff** [comment]
Fixed by #1060
